src/Server/PlayerAI/AppleAI.py

```diff
@@ -6,7 +6,7 @@
 class AppleAI(PlayerAILogic, AppleLogic):
     def __init__(self, player_id, terrain, game):
         super(AppleAI, self).__init__(player_id, terrain, game)
-        self._attack_range = 150  # TODO compute it normally
+        self._attack_range = 150
 
     def _attack(self):
         force_of_jump = self._calculate_attack_force(self.pos.x, self.pos.y + 100) # TODO egyelőre csak felfele ugrik egyet
```

Comment:
I can see an unresolved TODO over there :D

Reply:
"Some TODO remained but they are either solved on other branch or should be solved only right before finishing the project."